__tests__/help.ts

```diff
@@ -20,3 +21,15 @@ test('npx unimported --help', (done) => {
     done();
   });
 });
+
+test('npx unimported --help', async (done) => {
+  // note about `./` path: jest executes the tests from the root directory
+  // wrong file name
+  exec('node ./bin/1unimported.js --help', async (error, stdout, stderr) => {
+    expect(error).not.toBe(null);
+    expect(stderr).not.toBe(null);
+    expect(stdout).toBe('');
+
+    done();
+  });
```

Comment:
I think you misunderstood my last comment, I meant that I expected to see a test more like:

```ts
test('npx unimported --help', async () => {
  const stdout = await exec('node ./bin/1unimported.js --help');

  expect(stdout.trim()).toMatchInlineSnapshot(`
    "unimported

    scan your project for dead files
    Options:
      --version     Show version number                                    [boolean]
      --help        Show help                                              [boolean]
      --init, -i    dump default settings to .unimportedrc.json            [boolean]
      --flow, -f    indicates if your code is annotated with flow types    [boolean]
      --update, -u  update the ignore-lists stored in .unimportedrc.json   [boolean]"
  `);
});
```